### PR TITLE
Skeleton and masked mesh IO

### PR DESCRIPTION
**Pull request by ceesem**
Mostly IO of skeletons and masked meshes, with a few small changes here and there. I also added a file for useful basic mesh filters.

### CODE REVIEW

**Comment by sdorkenw** [comment]
Looks fine to me but appears to have some overlap with #21 in the VTK part

File: meshparty/mesh_filters.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    '''
+    cc, labels = sparse.csgraph.connected_components(mesh.csgraph, directed=False)
+    uids, counts = np.unique(labels, return_counts=True)
+    good_labels = uids[counts>size_thresh]
+    return np.in1d(labels, good_labels)
+
+
+def filter_small_component(mesh, size_thresh=1000):
+    '''
+    Returns a mesh filter without any connected components less than a size threshold
```
Comment (fcollman) [inline meshparty/mesh_filters.py]: doc string needs to be corrected, its a copy of filter larger right now.

File: meshparty/trimesh_io.py [hunk 2]
```diff
@@ -44,11 +44,18 @@ def read_mesh_h5(filename):
         else:
             mesh_edges = None
 
-    return vertices, faces, normals, mesh_edges
+        if masked_mesh:
```
Comment (fcollman) [inline meshparty/trimesh_io.py]: we need to resolve this with #21

File: meshparty/trimesh_io.py [hunk 3]
```diff
@@ -67,20 +74,27 @@ def write_mesh_h5(filename, vertices, faces,
...
-def read_mesh(filename):
+def read_mesh(filename, masked_mesh=False):
     """Reads a mesh's vertices, faces and normals from obj or h5 file"""
 
     if filename.endswith(".obj"):
         vertices, faces, normals = read_mesh_obj(filename)
         mesh_edges = None
     elif filename.endswith(".h5"):
-        vertices, faces, normals, mesh_edges = read_mesh_h5(filename)
+        mesh_data = read_mesh_h5(filename, masked_mesh)
```
Comment (fcollman) [inline meshparty/trimesh_io.py]: also needs resolving with #21

Comment (ceesem) [reply meshparty/trimesh_io.py]: Definitely. I'm going to wait until we've merged #21 and update.

**Comment by sdorkenw** [comment]
@ceesem #21 was merged

**Comment by fcollman** [comment]
I think that #22 includes the resolved merge conflicts and other fixes on top of what casey did here.

**Pull request merged**